README.md

```diff
@@ -2,7 +2,13 @@
 
 ## About
 
-Repository for the visualization of the [CC-Catalog](https://github.com/creativecommons/cccatalog) data.
+The landscape of licensed content is wide and varied. We have domains linking to other domains, different license types, and some metadata. This information is extracted from the Internet monthly by Common Crawl. It is fair to mention that we have 250 million works and growing! The goal of this project is to show the users how licensers are connected, their licensed content, and show how the licensing wave is expanding.
+
+A live demo of the project can be found in [2D](http://ec2-3-80-82-250.compute-1.amazonaws.com/) and in [3D](http://ec2-3-80-82-250.compute-1.amazonaws.com/visualization_3d.html)
+
+Each node in the graph below represents a unique domain. The size of the node represents the amount of CC licensed content present on that domain. The force of a link between two nodes (node A and node B) is given by the number of links node A has that references node B.
```

Comment:
I think this explanation fits better in the README of the data_processing folder, what do you think? I feel like this last paragraph is not well connected with the other ones.

Reply:
Ok, I'll remove this paragraph entirely, since the information already is being conveyed in the README of the data_processing folder